modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -186,6 +189,33 @@ Every processed record of the Batch Job Instance starts with the same variables
 In the On Complete phase, none of these variables (not event the original ones) are visible. Only the final result is available in this phase. Moreover, since the Batch Job Instance executes asynchronously from the rest of the flow, no variable set in either a Batch Step or the On Complete phase will be visible outside the Batch Scope.
 
 
+=== Scheduling Strategy
+
+A Mule application can contain several batch job definitions. Each definition 
+has its own scheduling strategy. A *Scheduling Strategy* enables you to control 
```

Comment:
Why are we boldfacing and init. capping the second instance of "scheduling strategy"?
This use of the term is general, what the thing is, not specific to a UI label.

Reply:
one is a general use of the term, the other for the component in the UI. I know it looks weird but follows the guidelines, I think...